Dockerfile

```diff
@@ -7,6 +7,9 @@ WORKDIR /borg
 ENTRYPOINT ["/usr/bin/borgctl"]
 CMD ["--help"]
 
+ARG IMAGE_VERSION
+ENV IMAGE_VERSION ${IMAGE_VERSION:-1.0.0}
```

Comment:
As I do a automatic build and `ARG` isn't supported currently on the <hub.docker.com> we need the correct versionnumber here.